tests/test_upo_uncoupled_ham.py

```diff
@@ -92,112 +102,112 @@ def test_differential_correction(self):
             self.assertLessEqual(hausd_dist, 1e-8)
 
 
-    def test_turning_point(self):
-        """Test the solution obtained using turning point"""
+    # def test_turning_point(self):
```

Comment:
Should this be commented?

Reply:
this version got pushed between tests, done.